wifiphisher/common/phishinghttp.py

```diff
@@ -7,6 +7,7 @@
 logging.getLogger('tornado.access').disabled = True
 logging.getLogger('tornado.general').disabled = True
 from wifiphisher.common.constants import *
+import wifiphisher.common.attacks as attacks
```

Comment:
Having all "attacks" in one single file will make things hard for us.

Instead, we want to have one module that will be the API. Each attack should be a different module leveraging this API.